pkg/db/errors.go

```diff
@@ -18,3 +19,18 @@ func isDialError(err error) bool {
 	netError := &net.OpError{}
 	return errors.As(err, &netError) && netError.Op == "dial"
 }
+
+func (d *dbTx) handleSQLError(err error, cmdType string, query string) error {
```

Comment:
can you move the implementation to `pkg/db/tx.go`

Reply:
Can you please explain the rational behind it?

Reply:
Sorry about that, I should have explained first. Usually, you keep the methods in the same file you define your receiver. It is easy to find the implementation this way. There are exceptions, and typically, they are baked by a file name convention - for example large routers or controllers where each method defines a request implementation - some choose to have a file per handler or group them by categories. In that case, you will find a common prefix or suffix file name.
The handleSQLError method is part of `dbTX` which is implemented in `tx.go`, I would not expect to find one of its method in `errors.go`.

Reply:
Thanks for the explanation